src/net.c

```diff
@@ -768,22 +728,24 @@ coap_send_message_type(coap_session_t *session, coap_pdu_t *request, unsigned ch
 }
 
 /**
- * Calculates the initial timeout based on the global CoAP transmission
- * parameters ACK_TIMEOUT, ACK_RANDOM_FACTOR, and COAP_TICKS_PER_SECOND.
- * The calculation requires ACK_TIMEOUT and ACK_RANDOM_FACTOR to be in
+ * Calculates the initial timeout based on the session CoAP transmission
+ * parameters 'ack_timeout', 'ack_random_timeout', and COAP_TICKS_PER_SECOND.
+ * The calculation requires 'ack_timeout' and 'ack_random_timeout' to be in
  * Qx.FRAC_BITS fixed point notation, whereas the passed parameter @p r
  * is interpreted as the fractional part of a Q0.MAX_BITS random value.
  *
+ * @param session session timeout is associated with
  * @param r  random value as fractional part of a Q0.MAX_BITS fixed point
  *           value
- * @return   COAP_TICKS_PER_SECOND * ACK_TIMEOUT * (1 + (ACK_RANDOM_FACTOR - 1) * r)
+ * @return   COAP_TICKS_PER_SECOND * 'ack_timeout' *
+ *           (1 + ('ack_random_timeout' - 1) * r)
```

Comment:
I wonder if this parameter should better be called `ack_random_factor` to be consistent?

```shell
./include/coap/net.h: * parameters 'ack_timeout', 'ack_random_timeout', and COAP_TICKS_PER_SECOND.
./include/coap/net.h: * The calculation requires 'ack_timeout' and 'ack_random_timeout' to be in
./include/coap/net.h: *           (1 + ('ack_random_timeout' - 1) * r)
./src/net.c: * parameters 'ack_timeout', 'ack_random_timeout', and COAP_TICKS_PER_SECOND.
./src/net.c: * The calculation requires 'ack_timeout' and 'ack_random_timeout' to be in
./src/net.c: *           (1 + ('ack_random_timeout' - 1) * r)
```

Reply:
Apologies - corrected now